ml-package-versions.yml

```diff
@@ -0,0 +1,179 @@
+sklearn:
+  package_info:
+    pip_release: "scikit-learn"
+    install_dev: |
+      pip install git+https://github.com/scikit-learn/scikit-learn.git@master
+
+  models:
+    minimum: "0.20.4"
+    maximum: "0.23.2"
+    run: |
+      pytest tests/sklearn/test_sklearn_model_export.py --large
+
+  autologging:
+    minimum: "0.20.4"
+    maximum: "0.23.2"
+    requirements: ["matplotlib"]
+    run: |
+      pytest tests/sklearn/test_sklearn_training_session.py --large
+      pytest tests/sklearn/test_sklearn_autolog.py --large
+
+pytorch:
+  package_info:
+    pip_release: "torch"
+    install_dev: |
+      pip install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+
+  models:
+    minimum: "1.4.0"
+    maximum: "1.7.0"
+    requirements:
+      # https://github.com/pytorch/vision#installation
+      "== 1.4.0": ["torchvision==0.5.0", "scikit-learn"]
+      "== 1.5.0": ["torchvision==0.6.0", "scikit-learn"]
+      "== 1.5.1": ["torchvision==0.6.1", "scikit-learn"]
+      "== 1.6.0": ["torchvision==0.7.0", "scikit-learn"]
```

Comment:
This seems to unfortunately imply that we're going to have to add `== 1.7.0` with a different version of `torchvision`. Would it instead be sufficient to unpin `torchvision` and trust the PyPI resolver to find an appropriate version of `torchvision` given the pinned version of `torch`?

Reply:
You're right. If we run `pip install 'torch==1.4.0' torchvision`, pip should automatically pick an appropriate version. I'll fix it.

Reply:
Fixed!